packages/web/src/common/store/pages/token-dashboard/addNewSplWallet.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  accountSelectors,
+  cacheActions,
+  Chain,
+  getContext,
+  Kind,
+  newUserMetadata,
+  tokenDashboardPageActions,
+  tokenDashboardPageSelectors,
+  walletActions
+} from '@audius/common'
+import { call, put, select } from 'typed-redux-saga'
+
+import { upgradeToCreator } from 'common/store/cache/users/sagas'
+import { requestConfirmation } from 'common/store/confirmer/actions'
+import { confirmTransaction } from 'common/store/confirmer/sagas'
+import { fetchServices } from 'common/store/service-selection/slice'
+import { waitForBackendAndAccount } from 'utils/sagaHelpers'
+
+import { getAccountMetadataCID } from './utils'
+
+const { getBalance } = walletActions
+
+const { getConfirmingWallet } = tokenDashboardPageSelectors
+const { updateWalletError, setWalletAddedConfirmed } = tokenDashboardPageActions
+
+const { getUserId, getAccountUser } = accountSelectors
+
+const CONNECT_WALLET_CONFIRMATION_UID = 'CONNECT_WALLET'
+
+const generalWalletErrorMessage =
+  'An error occured while connecting a wallet with your account.'
+
+export function* addNewSplWallet() {
+  const confirmingWallet = yield* select(getConfirmingWallet)
+  const { signature, wallet, balance } = confirmingWallet
+  console.log('adding new spl wallet', signature, wallet, balance)
```

Comment:
woglog

Reply:
lol sorry this whole file was meant to not be added :D since its all in web for now